modules/templates/helper.go

```diff
@@ -377,6 +377,10 @@ func NewFuncMap() []template.FuncMap {
 			return setting.MermaidMaxSourceCharacters
 		},
 		"QueryEscape": url.QueryEscape,
+		"BeautifyRef": func(ref string) string {
```

Comment:
`BeautifyRef` seems not that good.

1. It is only in a specialized context.
2. If we would see "branch changed from the-fix to the-fix" if it is actually changed from "sub1/the-fix" to "sub2/the-fix"

Reply:
Yeah sorry, the serializing shouldn't be done in the template - still figuring out what the correct wording should be https://github.com/go-gitea/gitea/pull/17672#issuecomment-970792761

Reply:
I think this can be removed now?